modules/Utilities.js

```diff
@@ -523,6 +524,20 @@ class Utilities {
         });
     }
 
+    static checkIfArangoIsRunning() {
```

Comment:
This method is not checking, it's more like checkOrFail() since there are not two different outcome.
My suggestion is to name this method to getArangoDbVersion() and don't assert status, just fail if not 200.
```
static getArangoDbVersion() {
        return new Promise((resolve, reject) => {
            request
                .get(`http://${process.env.DB_HOST}:${process.env.DB_PORT}/_api/version`)
                .auth(process.env.DB_USERNAME, process.env.DB_PASSWORD)
                .then((res) => {
                    if(res.status === 200) {
                          resolve(res.body); 
                    } else {
                          reject('Failed to contact DB');
                    }
                }).catch((err) => {
                    reject(err);
                });
        });
    }
```